### DISCUSSION TITLE
QFieldCloud self-hosted installation problems

### DISCUSSION THREAD

**Discussion by jfdarveau** [opengisch/QField#4815] [Q&A]
I installed QFieldCloud on Ubuntu Server 22.04.3. Everything seems to work correctly since the `docker compose exec app python manage.py status` commands returns:

> Everything` seems to work properly: {'geodb': 'ok', 'storage': 'ok'}

However, nothing is listening on port 8002. Also, running all the unit and functional tests throws some errors (see attached file).

I feel I'm pretty close to have a functional instance but there seems to be something I'm missing... Any help would be greatly appreciated.

[QFieldCloud_test_errors.txt](https://github.com/opengisch/QField/files/13518221/QFieldCloud_test_errors.txt)

**Comment by kachmul2004** [comment]
> I installed QFieldCloud on Ubuntu Server 22.04.3. Everything seems to work correctly since the `docker compose exec app python manage.py status` commands returns:
> 
> > Everything` seems to work properly: {'geodb': 'ok', 'storage': 'ok'}
> 
> However, nothing is listening on port 8002. Also, running all the unit and functional tests throws some errors (see attached file).
> 
> I feel I'm pretty close to have a functional instance but there seems to be something I'm missing... Any help would be greatly appreciated.
> 
> [QFieldCloud_test_errors.txt](https://github.com/opengisch/QField/files/13518221/QFieldCloud_test_errors.txt)
> 

Seems you are missing a pip package called "fiona"
